packages/docs/package.json

```diff
@@ -35,5 +34,9 @@
     "@types/react": "^17.0.10",
     "@types/react-dom": "^17.0.10",
     "typescript": "^4.5.4"
+  },
+  "peerDependencies": {
+    "react": ">=16.0.0",
+    "react-dom": ">=16.0.0"
   }
```

Comment:
If react & react dom are listed in the `dependencies` section, React 18 gets installed and causes errors in other parts of the monorepo